src/.nuget/optdata/ibcmerge.csproj

```diff
@@ -9,8 +9,24 @@
     <PackageReference Include="microsoft.dotnet.ibcmerge" Version="[$(IbcMergePackageVersion)]" Condition="'$(IbcMergePackageVersion)'!=''" />
   </ItemGroup>
 
+  <!--                                                                       -->
+  <!-- Task: DumpIbcMergePackageVersion                                       -->
+  <!--                                                                       -->
+  <!-- Notes:                                                                -->
+  <!--                                                                       -->
+  <!-- DumpIbcMergePackageVersion is used by build.sh and build.cmd to pass  -->
+  <!-- the version information to cmake. The task will write a file to the   -->
+  <!-- intermediates directory to be read back by build.cmd/sh. The path for -->
+  <!-- the file is IbcMergePackageVersionOutputFile                          -->
+  <!--                                                                       -->
   <Target Name="DumpIbcMergePackageVersion">
-    <Message Importance="high" Text="$(IbcMergePackageVersion)" />
+    <!-- Error if PgoDataPackageVersionOutputFile is not set. -->
+    <Error Condition="'$(IbcMergePackageVersionOutputFile)'==''" />
```

Comment:
Give the all errors a message indicating that the property <...> must be set. Otherwise, I think it just prints (No message specified) as the error message.